### PR TITLE
is related to: Organize base template structure - structure created, …

### PR DESCRIPTION
**Pull request by augustynSybidlo**
…emptypage deleted

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/template-types/newsportal-basepage/structure/.content.xml [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
-        cq:lastModified="{Date}2019-04-30T15:36:51.159+02:00"
+        cq:lastModified="{Date}2019-04-15T15:52:57.778+02:00"
         cq:lastModifiedBy="admin"
         jcr:primaryType="cq:PageContent"
         sling:resourceType="newsportal/components/structure/page">
-        <navigation
+        <header
             jcr:created="{Date}2016-01-13T15:58:16.080+01:00"
             jcr:createdBy="admin"
             jcr:lastModified="{Date}2016-01-13T15:58:16.080+01:00"
```
Comment (stgreenrecords) [inline ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/template-types/newsportal-basepage/structure/.content.xml]: in the ticke there were another points
I want to have the following templates created from the template-type:
news-template
blank-template

File: ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/template-types/newsportal-basepage/initial/.content.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="cq:PageContent"
+        sling:resourceType="newsportal/components/structure/page">
+        <header
```
Comment (stgreenrecords) [inline ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/template-types/newsportal-basepage/initial/.content.xml:7]: rename to navigation

File: ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/template-types/newsportal-basepage/structure/.content.xml [hunk 3]
```diff
@@ -3,11 +3,11 @@
     jcr:primaryType="cq:Page">
     <jcr:content
         cq:deviceGroups="[/etc/mobile/groups/responsive]"
-        cq:lastModified="{Date}2019-04-30T15:36:51.159+02:00"
+        cq:lastModified="{Date}2019-04-15T15:52:57.778+02:00"
         cq:lastModifiedBy="admin"
         jcr:primaryType="cq:PageContent"
         sling:resourceType="newsportal/components/structure/page">
-        <navigation
+        <header
```
Comment (stgreenrecords) [inline ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/template-types/newsportal-basepage/structure/.content.xml:8]: rename to navigation

File: ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/template-types/newsportal-basepage/initial/.content.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            jcr:created="{Date}2016-01-13T15:58:16.080+01:00"
```
Comment (stgreenrecords) [inline ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/template-types/newsportal-basepage/initial/.content.xml]: remove pls jcr:created, jcr:createdBy, jcr:lastModified,jcr:lastModifiedBy properties from all nodes

File: ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/templates/blank-template/.content.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Template">
+    <jcr:content
+        cq:lastModified="{Date}2019-05-20T09:37:07.106+02:00"
```
Comment (stgreenrecords) [inline ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/templates/blank-template/.content.xml]: remove pls all "meta" properties

Comment (augustynSybidlo) [reply ui.content/src/main/content/jcr_root/conf/newsportal/settings/wcm/templates/blank-template/.content.xml]: Done

**Pull request merged**